test/acceptance/test/install_tests.go

```diff
@@ -131,6 +153,78 @@ var _ = Describe("Weave GitOps Install Tests", func() {
 		})
 	})
 
+	It("Verify that gitops can install(via pull requests) & uninstall(via auto-merge) gitops components to multiple clusters under a user-specified namespace", func() {
```

Comment:
@josecordaz Please add skip tag like so:

![image](https://user-images.githubusercontent.com/76788247/148129341-9aa1fb72-3aba-4966-9bcf-ac9319c8df30.png)

Or else nightly tests will start failing. :(

Reply:
@rokshana-b  Sure, no problem. Is there any documentation where it explains where to add this skip?

Reply:
There isn't anything in the docs as of now - I'll add something in my PR (explaining the tag and when to use it)